rotkehlchen/tests/utils/ethereum.py

```diff
@@ -161,18 +162,9 @@ def setup_ethereum_transactions_test(
         one_receipt_in_db: bool = False,
         second_receipt_in_db: bool = False,
 ) -> tuple[list[EvmTransaction], list[EvmTxReceipt]]:
+    """This setup assummes that TEST_ADDR1 and TEST_ADDR2 are already present in the db"""
     dbevmtx = DBEvmTx(database)
     tx_hash1 = deserialize_evm_tx_hash('0x692f9a6083e905bdeca4f0293f3473d7a287260547f8cbccc38c5cb01591fcda')  # noqa: E501
-    addr2 = string_to_evm_address('0x442068F934BE670aDAb81242C87144a851d56d16')
```

Comment:
No, this change was because in the test `test_get_transaction_receipt` we were creating one extra account in the mocked ethereum object in addition to those checked in the test. I changed it to create explicitly only the addressed tested. This doesn't affect any other VCR test, only those using `setup_ethereum_transactions_test`